extensions/geography/description.yml

```
@@ -0,0 +1,22 @@
+extension:
+  name: geography
+  description: Global spatial data processing on the sphere
+  version: 0.1.0
+  language: C++
+  build: cmake
+  license: MIT
```

```suggestion
  license: MIT
  excluded_platforms: "wasm_mvp;wasm_eh;wasm_threads"
```
I believe wasm might need some extra options to actually work, I will come back on this.